src/views/CrontabForm/CrontabForm.tsx

```diff
@@ -0,0 +1,236 @@
+            <TextInput
+              id="crontab-cronSpec"
+              value={cronSpec || ""}
+              onChange={(e) =>
+                setCronSpec((e.target as HTMLInputElement).value)
+              }
+              // isRequired
+            />
+            <FormHelperText>
+              <HelperText>
+                <HelperTextItem>
+                  {t(
+                    "console-app~Defines the schedule on which the job will run (e.g., */5 * * * *)."
+                  )}
+                </HelperTextItem>
+              </HelperText>
+            </FormHelperText>
+          </FormGroup>
+
+          <FormGroup label="Image" fieldId="crontab-image" isRequired>
+            <TextInput
+              id="crontab-image"
+              value={image || ""}
+              onChange={(e) => setImage((e.target as HTMLInputElement).value)}
+              isRequired
+            />
+            <FormHelperText>
+              <HelperText>
+                <HelperTextItem>
+                  {t(
+                    "console-app~Specifies the container image to be executed by the CronTab."
+                  )}
+                </HelperTextItem>
+              </HelperText>
+            </FormHelperText>
+          </FormGroup>
+
+          <FormGroup label="Replicas" fieldId="crontab-replicas" isRequired>
+            <NumberInput
+              id="crontab-replicas"
+              value={replicas}
+              min={minValue}
+              max={maxValue}
+              onChange={onReplicasChange}
+              onMinus={onReplicasMinus}
+              onPlus={onReplicasPlus}
+              onBlur={onReplicasBlur}
+              inputName="replicas"
+              inputAriaLabel="number of replicas"
+              minusBtnAriaLabel="decrease replicas"
+              plusBtnAriaLabel="increase replicas"
+            />
+            <FormHelperText>
+              <HelperText>
+                <HelperTextItem>
+                  {t(
+                    "console-app~The desired number of instances of this CronTab to run."
+                  )}
+                </HelperTextItem>
+              </HelperText>
+            </FormHelperText>
+          </FormGroup>
+          {error && <Alert variant="danger" title={error} />}
+
+          <ActionGroup>
+            <Button
+              type="button"
+              variant="primary"
+              isDisabled={loading}
+              onClick={(e) => handleSubmit(e)}
+            >
+              {loading && <Spinner size="sm" className="pf-u-mr-sm" />}
+              {t("console-app~Create")}
+            </Button>
+            <Button
+              variant="secondary"
+              isDisabled={loading}
+              onClick={() =>
+                navigate(
+                  `/k8s/ns/${activeNamespace}/stable.example.com~v1~CronTab`
+                )
+              }
+            >
+              {t("console-app~Cancel")}
```

Comment:
I just noticed this on the last instance, but you'll need to update all instances of `console-app~` used as an i18n namespace:
```suggestion
              {t("console-crontab-plugin~Cancel")}
```